packer/ansible/roles/aggregate/files/aggregate-config.sh

```diff
@@ -49,13 +55,28 @@ showHelp() {
   echo "                     (not recommended if your IP address changes)"
   echo "--http-port  <value> Set a new HTTP port"
   echo "--https-port <value> Set a new HTTPS port"
+  echo "--net-mode   <value> Set the VM's network mode. Use 'nat' or 'bridge'"
 }
 
 if [ ${HELP} = YES ]; then
   showHelp
   exit 0
 fi
 
+if [ ${NET_MODE} != "nat" ] && [ ${NET_MODE} != "bridge" ]; then
```

Comment:
You'll get a unary operator expected error if you don't have a ${NET_MODE} set. The error will happen on line 66, 73, 111, 116.

Related, you might want to run the script through https://www.shellcheck.net to check for other issues.